src/locate/mod.rs

```diff
@@ -15,17 +15,27 @@ mod macos;
 #[cfg(target_os = "macos")]
 use crate::locate::macos::locate_steam_dir_helper;
 
+#[derive(Clone, Debug, Default)]
```

Comment:
```suggestion
#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

All the usual derives for a plain enum. I'm also leaning against having a `Default` impl for this since I'm not sure it makes much sense

Having this type be copy also means that `SteamDir::installation_type() -> &InstallationType` can return an owned `InstallationType` instead since it's a trivial copy